configs/ddrnet/ddrnet-23-slim_in1k-pre_2x8_1024x1024_160k_cityscapes.py

```diff
@@ -0,0 +1,16 @@
+_base_ = [
+    '../_base_/models/ddrnet.py',
+    '../_base_/datasets/cityscapes_1024x1024.py',
+    '../_base_/default_runtime.py', '../_base_/schedules/schedule_160k.py'
+]
+ 
+data = dict(
+    samples_per_gpu=8,
+    workers_per_gpu=8,
```

Comment:
```suggestion
    samples_per_gpu=4,
    workers_per_gpu=4,
```
To keep the same with former Cityscapes dataset config, we may set default setting 4 GPUs with 4 samplers per GPU.

Reply:
Sorry, i  just evaluate models on the Cityscapes val set. DDRNet-23-slim gains 77.37% mIoU  ,and DDRNet-23 gains 78.66% mIoU .Compared with  val results (77.8%,79.5%) from DDRNet repo ,i think results are acceptable,because some basic training tricks such as OHEM are not used.DDRNet-39 gains 79.46% mIoU on Cityscapes val set.